x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/service.test.ts

```diff
@@ -70,12 +81,56 @@ const TRUSTED_APP: TrustedApp = {
   ],
 };
 
+const TRUSTED_APP_BY_POLICY: TrustedApp = {
```

Comment:
Can you convert these constants to funcitons that return the payload, so that we avoid state between tests?